pkg/kademlia/routing_helpers_test.go

```diff
@@ -44,25 +55,29 @@ func newTestRoutingTable(localNode pb.Node) (*RoutingTable, error) {
 	return rt, nil
 }
 
-func createRoutingTable(t *testing.T, localNodeID storj.NodeID) (*RoutingTable, func()) {
+func createRoutingTableWith(localNodeID storj.NodeID, opts routingTableOpts) (*RoutingTable, func()) {
 	if localNodeID == (storj.NodeID{}) {
 		localNodeID = teststorj.NodeIDFromString("AA")
 	}
 	localNode := pb.Node{Id: localNodeID}
 
-	rt, err := newTestRoutingTable(localNode)
+	rt, err := newTestRoutingTable(localNode, opts)
 	if err != nil {
-		t.Fatal(err)
+		panic(err)
```

Comment:
what's the benefit of Fatal vs panic ?

Reply:
just that i don't have a *testing.T here anymore

Reply:
When running tests in parallel then panic will end up in a random location between the test results.

Reply:
ah, wait I might be confusing it with data-race / fmt.Print